Classes/Mailer.php

```diff
@@ -144,7 +144,9 @@ private function setTitle($htmlSrc)
     {
         // Extract title from HTML
         preg_match('|<title[^>]*>(.*)</title>|i', $htmlSrc, $m);
-        $title = trim($m[1]);
+
+        // As this is being extracted from HTML and it is being used as an email subject we need to decode any entities.
+        $title = trim(html_entity_decode($m[1]));
```

Comment:
This should probably be another PR, but most importantly it should be covered with tests.